SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -139,6 +150,9 @@ cargo build --release --target=%{rust_musl_target} %{cargo_pkg_feature_opts} %{c
 %license LICENSE-BSD-3-Clause
 
 %changelog
+* Tue Sep 17 2024 Jiri Appl <jiria@microsoft.com> - 38.0.72.2-3
```

Comment:
This doesn't seem to match the state on `fasttrack/2.0`, where Jiri's change was the 38.0.72.2-2 version. Please make sure that fast-track changes remain the same.

Reply:
updated